src/components/Header/Footer.tsx

```diff
@@ -0,0 +1,110 @@
+export const Footer = () => {
```

Comment:
can you rename the folder "Navigation" instead of "Header" so that the name is inclusive of the footer as well?